translations/system-configuration.de.xliff

```diff
@@ -184,7 +184,7 @@
             </trans-unit>
             <trans-unit id="Default">
                 <source>Default</source>
-                <target>Standard: Start werden nach oben, Ende und Dauer nach oben gerundet</target>
+                <target>Standard: Start werden nach unten, Ende und Dauer nach oben gerundet</target>
```

Comment:
`Start wird nach unten` sounds better, doesn't it?

Reply:
Yep. True!